lib/src/component_declaration/component_base.dart

```diff
@@ -808,3 +822,43 @@ class StateMeta implements AccessorMeta<StateDescriptor> {
 
   const StateMeta({this.fields, this.keys});
 }
+
+abstract class _AccessorMetaCollection<T extends _Descriptor, U extends AccessorMeta<T>> implements AccessorMeta<T> {
+  // todo probably make this public
```

Comment:
Did you want to address this in this PR?

Reply:
I'll just remove this comment

Reply:
Hahaha sounds good!